Sources/Common/Core/ScalarsToColors/index.d.ts

```diff
@@ -134,7 +134,7 @@ export interface vtkScalarsToColors extends vtkObject {
 	 * otherwise return isOpaque()
 	 * @see isOpaque, getAlpha
 	 */
-	areScalarsOpaque(scalars, colorMode, componentIn): boolean;
+	areScalarsOpaque(scalars: any, colorMode: number, componentIn: number): boolean;
```

Comment:
Thanks for typing this! `scalars` should be typed as `vtkDataArray`.

Reply:
And `colorMode` should be typed as [`ColorMode`](https://github.com/Kitware/vtk-js/blob/master/Sources/Rendering/Core/Mapper/Constants.d.ts#L1).

Reply:
should be resolved by now